code/game/bioscans.dm

```diff
@@ -112,12 +115,14 @@ GLOBAL_DATUM_INIT(bioscan_data, /datum/bioscan_data, new)
 /// The announcement to all Humans. Slightly off for the planet and elsewhere, accurate for the ship.
 /datum/bioscan_data/proc/ares_bioscan(forced = FALSE, variance = 2)
 	if(!forced && !can_ares_bioscan())
-		message_admins("An ARES Bioscan has failed.")
+		message_admins("BIOSCAN: An ARES bioscan has failed.")
 		return
-
 	//Adjust the randomness there so everyone gets the same thing
 	var/fake_xenos_on_planet = max(0, xenos_on_planet + rand(-variance, variance))
 
+	log_game("BIOSCAN: ARES bioscan completed.")
+	log_game("BIOSCAN: Bioscan complete. Sensors indicate [xenos_on_ship_uncontained ? "[xenos_on_ship_uncontained]" : "no"] unknown lifeform signature[!xenos_on_ship_uncontained || xenos_on_ship_uncontained > 1 ? "s":""] present on the ship[xenos_on_ship_uncontained && xenos_ship_location ? ", including one in [xenos_ship_location]," : ""] and [fake_xenos_on_planet ? "approximately [fake_xenos_on_planet]" : "no"] signature[!fake_xenos_on_planet || fake_xenos_on_planet > 1 ? "s":""] located elsewhere[fake_xenos_on_planet && xenos_planet_location ? ", including one in [xenos_planet_location]":""].")
```

Comment:
It would be better to just have a string that both the log and the official message can share. This is almost exactly the var/input below just without a preceding "BIOSCAN: " and new line usage.